charts/renku/templates/secrets.yaml

```diff
@@ -19,7 +19,8 @@ data:
   gitlab-password: {{ .Values.global.gitlab.postgresPassword | b64enc | quote }}
 {{- end }}
 {{- if .Values.keycloak.enabled }}
-  keycloak-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
+  keycloak-password: {{ .Values.keycloak.keycloak.password | b64enc | quote }}
+  keycloak-postgres-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
   renku-realm.json: {{ include "renku.keycloak.realm" . | indent 4  | b64enc | quote }}
 {{- end }}
   jupyterhub-password: {{ .Values.global.jupyterhub.postgresPassword | b64enc | quote }}
```

Comment:
should we change the other postgress passwords?

Reply:
good idea, would make sense imo

Reply:
@leafty, @rokroskar : ping! what's the action point here?

Reply:
for this particular comment -- rename the other password variable names to include "postgres" just as was done with keycloak